simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -107,7 +112,8 @@ export const ExtraInfo = () => (
       <img src="/images/easy.svg" alt="easy to use" />
       <p className={styles['brief-description']}>Easy to use</p>
       <p className={styles['detailed-description']}>
-        We provide free queue management thereby allowing you to focus on better things
+        We provide free queue management service and help you focus on important aspects of your
+        activity
       </p>
```

Comment:
How about "We provide free queue management thereby allowing you to focus on other things"?